test/e2e/tests/account/incremental-security.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { withFixtures, WALLET_PASSWORD } from '../../helpers';
```

Comment:
related to my 1st comment, I would then import this from the constants file `WALLET_PASSWORD`